src/CoreWCF.UnixDomainSocket/src/CoreWCF/Channels/UnixDomainSocketHostedService.cs

```diff
@@ -41,7 +41,7 @@ public UnixDomainSocketHostedService(IServiceBuilder serviceBuilder, ILogger<Uni
         private void Init()
         {
             // Check if Kestrel is already being used and if it is, it was already configured so there's nothing to do.
-            var unixDomainSocketFramingOptionsSetup = _serviceProvider.GetServices<IConfigureOptions<KestrelServerOptions>>().SingleOrDefault(options => options is UnixDomainSocketFramingOptionsSetup) as UnixDomainSocketFramingOptionsSetup;
+          //  var unixDomainSocketFramingOptionsSetup = _serviceProvider.GetServices<IConfigureOptions<KestrelServerOptions>>().SingleOrDefault(options => options is UnixDomainSocketFramingOptionsSetup) as UnixDomainSocketFramingOptionsSetup;
```

Comment:
>          //  var unixDomainSocketFramingOptionsSetup = _serviceProvider.GetServices<IConfigureOptions<KestrelServerOptions>>().SingleOrDefault(options => options is UnixDomainSocketFramingOptionsSetup) as  [](http://example.com/codeflow?start=0&length=206)

Remove instead of leaving commented out. You should be able to remove this entire method and remove the call to it.